spec/features/homepage_text_spec.rb

```diff
@@ -26,7 +26,7 @@
 
   xit 'includes a link to the dev portal home page in the navigation' do
     within '.navbar' do
-      expect(page).to have_link 'CHARMcare Data Developers', href: root_path
+      expect(page).to have_link 'CHARMcare', href: root_path
```

Comment:
This test doesn't run, I wonder if it makes sense to have it run and check that this works?